src/build.rs

```diff
@@ -21,8 +57,34 @@ fn main() {
     #[cfg(feature = "no_bssl")]
     return;
 
+    let arch = std::env::var("CARGO_CFG_TARGET_ARCH").unwrap();
+    let os = std::env::var("CARGO_CFG_TARGET_OS").unwrap();
+
+    // It doesn't execute cmake until .build(), so initialize here
+    let mut boringssl_cmake = cmake::Config::new("deps/boringssl");
```

Comment:
Maybe we could wrap up creation of `boringssl_cmake` in a function that does os detection returns a `cmake::Config`. That way, `main()` stays quite small and concise.

Reply:
Ok. I added 939ccdc. Let me know what you think. If this works ok I will update ios (#71) as well.